src/utils/utils.ts

```diff
@@ -0,0 +1,13 @@
+export interface Body {
+  id: number;
+  name: string;
+  email: string;
+  password: string;
+}
+
+export const errorCode = {
+  NOT_FOUND: 'NOT_FOUND',
+  INVALID: 'INVALID',
+  CONNECTION: 'CONNECTION',
+  INTERNAL_SERVER_ERROR: 'INTERNAL_SERVER_ERROR',
+};
```

Comment:
Please add `as const`, here is a reason why
https://www.youtube.com/watch?v=6M9aZzm-kEc